### DISCUSSION TITLE
Feature Request on `flux:input` with inputmode

### DISCUSSION THREAD

**Discussion by littpi** [livewire/flux#512] [Feature Requests] [answered]
## Background
In mobile view, sometimes we need to use custom keyboard for different approaches. Like numpad for telephone, or a keyword with `@` for easily input email. In fact, there is an attribute named [inputmode](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode) and I think many frontend devs are familiar with it.

## Suggestion
I don't know if I am doing it wrong, but it seems there are no inputmode attribute inside the blade file for the input. Therefore I suggest to add inputmode for `flux:input` if it is not implemented.

**Comment by jeffchown** [comment]
@littpi Based on https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode and the repeated mentions of the `<input type="..." />`, aren't most cases/keyboards covered when we properly set the `type`?  e.g. `<flux:input type="number" />`, `<flux:input type="tel" />`, etc.

**Reply by littpi** [reply]
I just use to write inputmode because the browser seems not always supporting change the keyword before. I have tried using only `type` and it works now so I think it is time to say goodbye to `inputmode` perhaps.

**Comment by joshhanley** [comment, marked as answer]
@littpi you should already be able to use input mode, try `<flux:input inputmodel="email" />`

**Reply by littpi** [reply]
Thanks for the help. I can use it now after I updated the package. It is weird that I cannot use it beforehand. 😩
